packages/block-library/src/media-text/transforms.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * External dependencies
+ */
+import { set } from 'lodash';
```

Comment:
My understanding is there's a general preference for reducing our use of Lodash, with the view to one day drop it entirely. I think we can avoid the use of `set` here.